Source/Scene/GroundPrimitive.js

```diff
@@ -0,0 +1,655 @@
+        StencilOperation) {
+    "use strict";
+
+    /**
+     * A ground primitive represents geometry draped over the terrain in the {@link Scene}.  The geometry must be from a single {@link GeometryInstance}.
+     * Batching multiple geometries is not yet supported.
+     * <p>
+     * A primitive combines geometry instances with an {@link Appearance} that describes the full shading, including
+     * {@link Material} and {@link RenderState}.  Roughly, the geometry instance defines the structure and placement,
+     * and the appearance defines the visual characteristics.  Decoupling geometry and appearance allows us to mix
+     * and match most of them and add a new geometry or appearance independently of each other. Only the {@link PerInstanceColorAppearance}
+     * is supported at this time.
+     * </p>
+     *
+     * @alias GroundPrimitive
+     * @constructor
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {Array|GeometryInstance} [options.geometryInstances] A single geometry instance to render.
+     * @param {Boolean} [options.show=true] Determines if this primitive will be shown.
+     * @param {Boolean} [options.vertexCacheOptimize=false] When <code>true</code>, geometry vertices are optimized for the pre and post-vertex-shader caches.
+     * @param {Boolean} [options.interleave=false] When <code>true</code>, geometry vertex attributes are interleaved, which can slightly improve rendering performance but increases load time.
+     * @param {Boolean} [options.compressVertices=true] When <code>true</code>, the geometry vertices are compressed, which will save memory.
+     * @param {Boolean} [options.releaseGeometryInstances=true] When <code>true</code>, the primitive does not keep a reference to the input <code>geometryInstances</code> to save memory.
+     * @param {Boolean} [options.allowPicking=true] When <code>true</code>, each geometry instance will only be pickable with {@link Scene#pick}.  When <code>false</code>, GPU memory is saved.
+     * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready.
+     * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Determines if this primitive's commands' bounding spheres are shown.
+     *
+     * @see Primitive
+     * @see GeometryInstance
+     * @see Appearance
+     *
+     * @example
+     * var rectangleInstance = new Cesium.GeometryInstance({
+     *   geometry : new Cesium.RectangleGeometry({
+     *     rectangle : Cesium.Rectangle.fromDegrees(-140.0, 30.0, -100.0, 40.0)
+     *   }),
+     *   id : 'rectangle',
+     *   attributes : {
+     *     color : new Cesium.ColorGeometryInstanceAttribute(0.0, 1.0, 1.0, 0.5)
+     *   }
+     * });
+     * scene.primitives.add(new Cesium.GroundPrimitive({
+     *   geometryInstances : rectangleInstance
+     * }));
+     */
+    var GroundPrimitive = function(options) {
```

Comment:
I know you want to make a pass on the doc, but we should document here what geometries are compatible with it, and also mention it in each geometry.